.travis.yml

```diff
@@ -1,14 +1,18 @@
 language: php
 
 php:
-    - 5.4
-    - 5.5
     - 5.6
     - 7.0
     - 7.1
     - hhvm
 
 before_script: composer install --dev --prefer-source
 
+script:
+    - vendor/bin/phpunit -v
+
+dist: trusty
+sudo: required
```

Comment:
Fair enough... be good to sort the travis bundles to build low & high dependencies also (not for this PR).